docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -1121,6 +1121,18 @@ the remote end.
 include::../../../gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Docs/Reference/GremlinVariantsTests.cs[tags=connecting]
 ----
 
+Some connection options can also be set on individual requests made through the using `With()` step on the
+`TraversalSource`. For instance to set request timeout to 500 milliseconds:
+
+[source,csharp]
+----
+var l = g.With(Tokens.ArgsEvalTimeout, 500).V().Out("knows").Count();
```

Comment:
I'm not sure about this but shouldn't we use a terminal step here to actually get the count(s)? Especially new users would probably expect here to have the count stored in the variable and not a traversal object.